src/WarcraftLegacies.Source/Quests/Dalaran/QuestDalaran.cs

```diff
@@ -19,12 +19,14 @@ namespace WarcraftLegacies.Source.Quests.Dalaran
   public sealed class QuestDalaran : QuestData
   {
     private readonly List<unit> _rescueUnits = new();
+    private readonly unit _gilneasDoor;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="QuestDalaran"/> class.
     /// </summary>
     /// <param name="rescueRects">Units inside these rectangles start invulnerable and are rescued when the quest ends.</param>
     /// <param name="prerequisites">These quests must be completed before this one can be completed.</param>
+    /// /// <param name="gilneasDoor">This unit will be transferred to the completeing player.</param>
```

Comment:
Need to clean this up since it's not used in this quest anymore. Same thing for line 22.
Note that the compiler actually warned us about this in the Github Action, i.e: 
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/8995690/5ec0bb6a-bb56-4fc7-b270-f7eacdf6759f)